CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+* Add `index-of` and `slice` expressions to search arrays and strings for the first occurrence of a specified value and return a section of the original array or string ([#9450](https://github.com/mapbox/mapbox-gl-js/pull/9450)) (h/t [lbutler](https://github.com/lbutler))
+* Correctly set RTL text plugin status if the plugin URL could not be loaded. This allows developers to add retry logic on network errors when loading the plugin ([#9489](https://github.com/mapbox/mapbox-gl-js/pull/9489))
+
+### 🍏 Improvements
+
+* An outstanding list of gesture issues and improvements were addressed in this release ([#9365](https://github.com/mapbox/mapbox-gl-js/pull/9365))
+    - You can now interact with multiple maps on the same page at the same page. For example, pinch rotate one map while touch panning a different one
+    - All gestures now have inertial. Previously touch rotation and mouse pitching did not have any
+    - It is now possible to switch fingers while panning the map without interrupting the pan. While panning: add a second finger, keep panning, remove the first, keep panning
+    - Touch rotation thresholding is improvement to make it harder to accidentally rotate when your fingers are very close together
+    - Fix map jump when releasing one finger after pinch zoom ([#9136](https://github.com/mapbox/mapbox-gl-js/issues/9136))
+    - Stop mousedown and touchstart from interrupting easeTo animations if interaction handlers are disabled ([#8725](https://github.com/mapbox/mapbox-gl-js/issues/8725))
+    - Stop mouse wheel from interrupting animations when `map.scrollZoom` is disabled ([#9230](https://github.com/mapbox/mapbox-gl-js/issues/9230))
+    - Fix `map.isZooming()` and `map.isMoving()` while zooming with Apple Magic mouse ([#6562](https://github.com/mapbox/mapbox-gl-js/issues/6562))
+  - _Changes to undocumented behavior_
+     - The previous gestures implementation prevented a camera change from actually happening if the interaction handler responsible for the change was disabled within the camera change event (`dragstart` or `rotatestart`). This behavior was not documented. It is no longer possible to prevent a camera change after it's event has been fired. It is possible some applications may be depending on this behavior. The correct way for these applications to selectively prevent camera changes is to listen to the `mousedown` or `touchstart` map event and call [.preventDefault()](https://docs.mapbox.com/mapbox-gl-js/api/#mapmouseevent#preventdefault) on it. For example, `map.on("mousedown", function(mapMouseEvent) { mapMouseEvent.preventDefault(); })`;
+     - The camera change events fired by the doubleClickZoom handler (`zoomstart`, `zoom`, `zoomend`, `movestart`, `move`, `moveend`) used to have undocumented properties such as `lngLat` and `point`. These have been removed
+* More precise collision detection for line labels ([#9219](https://github.com/mapbox/mapbox-gl-js/pull/9219))
```

Comment:
I appreciate the greater clarity that comes from breaking out this PR into sub-bullets, but given how long this changelog already is from the gestures improvement, I wonder whether we can shorten the syntax of some of the improvements and keep things as a single entry, e.g.:

> Line labels now have improved collision detection, with greater precision in placement, reduced memory footprint, better placement under pitched camera orientations.

Reply:
Updated in https://github.com/mapbox/mapbox-gl-js/pull/9542/commits/a1219354822fe417ce65bf1abf8377966f6a41a3